distributions/docker/replication/src/main/docker/files.xml

```diff
@@ -20,7 +20,7 @@
   <dependencySets>
     <dependencySet>
       <includes>
-        <include>*</include>
+        <include>com.connexta.ion.replication.distributions:replication-distros-spring</include>
       </includes>
       <outputDirectory>.</outputDirectory>
       <useTransitiveDependencies>false</useTransitiveDependencies>
```

Comment:
Below this where we copy resources, is there a reason that we copy the demo keystore/truststore over to the target directory? I was just thinking of deleting this assembly file.

Reply:
We copy them over so the docker image will work "out of the box" if you don't configure anything externally.